src/main/java/com/launchdarkly/client/EvaluationError.java

```diff
@@ -0,0 +1,10 @@
+package com.launchdarkly.client;
+
+/**
+ * An error indicating an abnormal result from evaluating a feature
+ */
+public class EvaluationError extends Exception {
```

Comment:
This should not be public- we don't throw it from any public method. 
Changing the name to EvaluationException is more consistent with Java naming conventions, unless we ARE going to throw this from a public method (probly not  a good idea), then we should name it LDEvaluationException or similar.